parsers/jscript.c

```diff
@@ -2398,10 +2415,47 @@ static bool parsePrototype (tokenInfo *const name, tokenInfo *const token, state
 					skipArgumentList(method_body_token, false,
 									 vStringLength (signature) == 0 ? signature : NULL);
 				else
+				{
+					if ( isType (method_body_token, TOKEN_EQUAL_SIGN) &&
+						! isType (identifier_token, TOKEN_UNDEFINED) &&
+						prototype_token_count < sizeof(prototype_tokens)/sizeof(tokenInfo*) - 1)
```

Comment:
You can use ARRAY_SIZE macro.

Reply:
No longer relevant after switching to ptrArray